datasets/ONS_CPA/main.py

```diff
@@ -6,196 +6,63 @@
 #       extension: .py
 #       format_name: light
 #       format_version: '1.5'
-#       jupytext_version: 1.4.2
+#       jupytext_version: 1.6.0
 #   kernelspec:
 #     display_name: Python 3
 #     language: python
 #     name: python3
 # ---
 
-# ###  CPA xlsx to Tidydata
-
-# +
 from gssutils import *
-
-scraper = Scraper('https://www.ons.gov.uk/businessindustryandtrade/internationaltrade/datasets/' + \
-                  'uktradeingoodsbyclassificationofproductbyactivity')
-scraper
+import json 
+
+# + tags=["outputPrepend"]
+info = json.load(open('info.json'))
+scraper = Scraper(seed='info.json')  
+cubes = Cubes('info.json')
+scraper.dataset.family = info['families']
+scraper 
 # -
 
-tab = scraper.distributions[1].as_pandas(header=None, na_values=[], keep_default_na=False)
-
-tab.rename(columns=tab.iloc[1], inplace=True)
-tab.rename(columns={'CDID': 'Period'}, inplace=True)
-tab
-
-# The observations are in rows 7 on.
-#
-# Each CDID corresponds to a unique time-series slice. Unpivot the table so we have one row per observation and drop any rows with no value for the observation.
-
-observations = tab[7:].rename(columns={'CDID': 'Period'})
-observations = pd.melt(observations, id_vars=['Period'], var_name='CDID', value_name='Value')
-observations['Value'] = pd.to_numeric(observations['Value'])
-observations.dropna(inplace=True)
-observations.reset_index(drop=True, inplace=True)
-observations
-
-# The date/time values need to be in a format that can be used to create URIs for British calendar intervals,
-# see https://github.com/epimorphics/IntervalServer/blob/master/interval-uris.md#british-calendar-intervals
+df = scraper.distributions[0].as_pandas(header=None, na_values=[], keep_default_na=False, index_col=0)
```

Comment:
could pass `scraper.distribution(latest=True)` instead of indexing it, it is more legacy code.

Reply:
Done